Lombiq.Hosting.Tenants.EmailQuotaManagement/Views/EmailTemplate__EmailQuote.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@{
+    ViewLayout = "Layout__EmailTemplate";
+}
+
+<p>@T["Hi,"]</p>
+<p>
+    @T["It seems that your {0} site sent out more e-mails than the available quota for this month. E-mail sending has " +
+        "been stopped until next month (any feature sending out e-mails will fail).", Model.HostName]
+</p>
+<p>
+    @T["Thank you,"]<br/>
+    @T["Your Admin Crew"]
```

Comment:
Fragments like this can't be properly localized. This should rather be a single localized string, including formatting. Also, the name of the file is "quote".